docs/notebooks/tutorials/advanced_optimization.ipynb

```diff
@@ -1222,7 +1222,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.11"
+   "version": "3.8.10"
```

Comment:
Is this change necessary?

Reply:
that is just a byproduct of jupyter, this is not a real change

Reply:
So why is it included in this PR? It's fine, but in the future don't add extra files if you don't need to

Reply:
will do